app/scripts/controllers/showGroupModal.js

```diff
@@ -0,0 +1,30 @@
+
+angular.module('confRegistrationWebApp')
+  .controller('showGroupModalCtrl', function($scope, conference, currentRegistration, registrationId){
+    $scope.currentRegistration = currentRegistration;
+    $scope.registrationId = registrationId;
+    $scope.conference = angular.copy(conference);
+
+    $scope.visibleRegistrantTypes = angular.copy($scope.conference.registrantTypes);
+
+    _.remove($scope.visibleRegistrantTypes, function(registrantType) {
+      //remove if type is marked as hidden and a registrant with this type doesn't already exist in the registration
+      return registrantType.hidden && !_.includes(_.map($scope.currentRegistration.registrants, 'registrantTypeId'), registrantType.id);
+    });
+
+    $scope.registrationTypeFull = function(type){
+      if(!type.useLimit){
+        return false;
+      }
+      if(!type.availableSlots){
+        return true;
+      }
+
+      // subtract currentRegistration's registrants from availableSlots. if there's no slot left, return true
+      if(type.availableSlots - _.filter($scope.currentRegistration.registrants, { 'registrantTypeId': type.id }).length <= 0){
```

Comment:
I still think this line could be cleaned up. It does extra math just to compare it against 0 when it could be compared directly and the return true/false are duplicate. You could just return the result of the comparison which is a bool.